app/pages/project/classify.cjsx

```diff
@@ -120,7 +121,7 @@ module.exports = React.createClass
         @setState {projectIsComplete}
         randomIndex = Math.floor Math.random() * workflows.length
         # console.log 'Chose random workflow', workflows[randomIndex].id
-        workflows[randomIndex]
+        @getWorkflow project, workflows[randomIndex].id
```

Comment:
This change is not necessary. All the workflows were already fetched and this just needs to return a random one out of the response array. It doesn't need to be fetched again.